src/landing.tsx

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+
+import jeOctopus from '../style/icons/landing/je-octopus.svg';
+import octopusOutline from '../style/icons/landing/je-octopus-outline.svg';
+import jupyterEverywhereText from '../style/icons/landing/jupytereverywhere-text.svg';
+
+import pythonLogo from '../style/icons/landing/python.svg';
+import rLogo from '../style/icons/landing/r.svg';
+
+import scrolldownArrow from '../style/icons/landing/scrolldown-arrow.svg';
+
+import codeSymbol from '../style/icons/landing/code.svg';
+import shareIcon from '../style/icons/landing/share.svg';
+import jupyterLogo from '../style/icons/landing/jupyter.svg';
+
+import testimonialAvatar from '../style/icons/landing/testimonial.svg';
+
+// TODO: get SVGs for these logos
+// import courseKataLogo from '../style/icons/landing/coursekata.svg';
+// import skewTheScriptLogo from '../style/icons/landing/skewthescript.svg';
+
+// TODO: find a better way to handle these imports
+import '../style/landing.css';
+import '../style/base.css';
+
+/**
+ * LandingPage component renders the landing page for Jupyter Everywhere.
+ * It includes a header, hero section with buttons to create notebooks,
+ * a features section, a testimonial, an about section, and a footer.
+ * @returns Landing page component
+ */
+function LandingPage(): JSX.Element {
+  const handleCreateNotebook = (kernel: string) => {
+    window.location.href = `/?notebook=Untitled.ipynb&kernel=${kernel}`;
+  };
+
+  // Placeholder function for upload functionality
+  const handleUpload = () => {
+    alert('Upload functionality not yet implemented.');
+  };
+
+  const handleScrollToFeatures = (e: React.MouseEvent<HTMLAnchorElement>) => {
+    e.preventDefault();
+    const featuresSection = document.getElementById('features');
+    if (featuresSection) {
+      featuresSection.scrollIntoView({
+        behavior: 'smooth',
+        block: 'start'
+      });
+    }
+  };
+
+  const handleCardKeyDown = (e: React.KeyboardEvent, kernel: string) => {
+    if (e.key === 'Enter' || e.key === ' ') {
+      e.preventDefault();
+      handleCreateNotebook(kernel);
+    }
+  };
+
+  return (
+    <div className="je-landing">
+      {/* Hero section */}
+      <section className="je-hero">
+        <div className="je-header">
+          <img src={jeOctopus} className="je-logo" alt="Logo" />
+          <button className="je-tutorial-button">Tutorial</button>
+        </div>
+
+        <main className="je-main">
+          <h2 className="je-brand">Welcome to</h2>
+          <h1>
+            <span className="je-brand">Jupyter Everywhere</span>
+          </h1>
```

Comment:
This will fail WCAG. It is a single header, I would suggest to make it all `h1` and just use spans for styling/font size adjustment.
https://www.w3.org/WAI/tutorials/page-structure/headings/